pkg/ccl/changefeedccl/helpers_test.go

```diff
@@ -676,6 +580,226 @@ func closeFeed(t testing.TB, f cdctest.TestFeed) {
 	}
 }
 
+// TestServer is a struct to allow tests to operate on a shared API regardless
+// of a test running as the system tenant or a secondary tenant, while still
+// having access to system interfaces for operations that require them such as
+// desctestutils.TestingGetPublicTableDescriptor
```

Comment:
we do have `testServerShim` is this a replacement for that?

Reply:
Do you want to delete testServerShim?

Reply:
Yep its already removed, the `helpers_test_shim_test.go` file is deleted completely